build_conf.py

```diff
@@ -218,6 +222,10 @@ def __parse_args(self):
         parser.add_argument('--prod_pulls',
                             dest='prod_pulls', required=False, default=False,
                             help="Use to define the list of the pull-requests for product's meta layer.")
+        parser.add_argument('--update-build',action='store_true',
+                            dest='update_build', required=False, default=False,
+                            help='Generte update bundle to perform SOTA/FOAT update')
```

Comment:
As I understand, we have two misprints here:
- Generte -> Generate
- FOAT -> FOTA